VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
```

Comment:
> and perhaps why it's not under .vs/

\*sigh\* yeah, I'd hoped that the `.vs/` directory would end the proliferation of files...